Hunting Queries/BehaviorAnalytics/Anomalous Account Manipulation.yaml

```diff
@@ -0,0 +1,33 @@
+id: 8741deeb-332e-4061-8873-5086040920e3
+name: Anomalous Account Manipulation
+description: |
+  'All high Blast Radius users performing Update user (name change) to priveleged role, or ones that changed users for the first time.'
+requiredDataConnectors:
+  - connectorId: BehaviorAnalytics
+    dataTypes:
+      - BehaviorAnalytics
+      - AuditLogs
+tactics:
+  - Persistence
+relevantTechniques:
+  - T1098
+query: |
+
+ let critical = dynamic(['9b895d92-2cd3-44c7-9d02-a6ac2d5ea5c3','c4e39bd9-1100-46d3-8c65-fb160da0071f','158c047a-c907-4556-b7ef-446551a6b5f7','62e90394-69f5-4237-9190-012177145e10','d29b2b05-8046-44ba-8758-1e26182fcf32','729827e3-9c14-49f7-bb1b-9608f156bbb8','966707d0-3269-4727-9be2-8c3a10f19b9d','194ae4cb-b126-40b2-bd5b-6091b380977d','fe930be7-5e62-47db-91af-98c3a49a38b1']);
+ let high = dynamic(['cf1c38e5-3621-4004-a7cb-879624dced7c','7495fdc4-34c4-4d15-a289-98788ce399fd','aaf43236-0c0d-4d5f-883a-6955382ac081','3edaf663-341e-4475-9f94-5c398ef6c070','7698a772-787b-4ac8-901f-60d6b08affd2','b1be1c3e-b65d-4f19-8427-f6fa0d97feb9','9f06204d-73c1-4d4c-880a-6edb90606fd8','29232cdf-9323-42fd-ade2-1d097af3e4de','be2f45a1-457d-42af-a067-6ec1fa63bc45','7be44c8a-adaf-4e2a-84d6-ab2649e08a13','e8611ab8-c189-46e8-94e1-60213ab1f814']);
+ AuditLogs
+ | where OperationName == "Update user"
+ | mv-expand AdditionalDetails
+ | mv-expand TargetResources
+ | where AdditionalDetails.key == "UserPrincipalName"
+ | mv-expand TargetResources
+ | extend RoleId = tostring(TargetResources.modifiedProperties[0].newValue)
+ | extend RoleName = tostring(TargetResources.modifiedProperties[1].newValue)
+ | where RoleId in (critical,high)
+ | extend TargetId = tostring(TargetResources.id)
+ | extend Target = tostring(TargetResources.userPrincipalName)
+ | where isnotempty(RoleId) or isnotempty(RoleName)

```

Comment:
For performance, maybe move this line up in the processing order

Reply:
done